impl/proto/keytransparency_v1_service/keytransparency_v1_service.proto

```diff
@@ -60,4 +63,45 @@ service KeyTransparencyService {
       body: "entry_update"
     };
   }
+
+  // GetMutationsByEpoch returns a list of mutations paged by epoch number.
+  //
+  // Returns a list of mutations and their inclusion proofs along with the epoch
+  // signed map root.
+
+  rpc GetMutationsByEpoch(keytransparency.v1.types.GetMutationsRequest)
```

Comment:
Is there a strong reason to have two APIs that do similar things? 
There might be an argument for simplicity to only have one.